assets/css/admin-notices.scss

```
@@ -7,10 +11,22 @@
 
 	// Spacing.
 	gap: 10px;
-	padding: 18px 24px;
+	padding: 10px 12px;
+
+	&.wpjm-admin-notice--error {
```

Just a suggestion to make use of the sass features, keeping it a little cleaner. The same applies to the next classes. ;)

```suggestion
	&--error {
```

---

Committed https://github.com/Automattic/WP-Job-Manager/pull/2401/commits/56386a5274e65a9c5abae0ac8e5e8f7ff62b1401